api/websocket.go

```diff
@@ -36,6 +37,41 @@ var upgrader = websocket.Upgrader{
 	WriteBufferSize: 1024,
 }
 
+// type safeWebsocketConn wraps websocket.Conn with a mutex
```

Comment:
the link could be added here 
https://pkg.go.dev/github.com/gorilla/websocket#hdr-Concurrency